swig/python/__init__.py.in

```diff
@@ -31,7 +31,15 @@ if os.name == 'nt' and hasattr(os, 'add_dll_directory') and callable(getattr(os,
 # module version
 __version__ = '@FLAMEGPU_VERSION_PYTHON@'
 
-del os, sys, pathlib, subprocess
 # Normal module stuff
 __all__ = ["@PYTHON_MODULE_NAME@"]
-from .@PYTHON_MODULE_NAME@ import *
\ No newline at end of file
+from .@PYTHON_MODULE_NAME@ import *
+
+# GLM delayed so we can check whether it was enabled
+if GLM:
+    if not "GLM_INC_DIR" in os.environ or not "GLM_INC_DIR" in os.environ:
+        os.environ["GLM_INC_DIR"] = str(pathlib.Path(__file__).resolve().parent / "glm")
+    else:
+      print("@PYTHON_MODULE_NAME@ warning: env var 'GLM_INC_DIR' is present, GLM include path may be incorrect.", file=sys.stderr)
```

Comment:
Should this really warn if the variable is set as soon as the module is imported? Feels very noisy if they are having to set it anyway. 

Also needs prefixing if the warning is kept. 

```
GLM_INC_DIR=/tmp/ python3
Python 3.10.6 (main, May 29 2023, 11:10:38) [GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pyflamegpu
pyflamegpu warning: env var 'GLM_INC_DIR' is present, GLM include path may be incorrect.
```